config/cpu_default.yml

```diff
@@ -1,6 +1,6 @@
 input_dir: ../labeled_images/
 output: result
-batch_size: 32
+batch_size: 3
```

Comment:
ehm, you sure you want to merge this line, too?

Reply:
Fixed